src/pages/index.jsx

```diff
@@ -9,10 +9,10 @@ export default function Home() {
           Welcome to the Accessibility Capstone!
         </h1>
         <ul>
-          <li><Link href='/accordion'><a>Accordion</a></Link></li>
-          <li><Link href='/dialog'><a>Dialog</a></Link></li>
-          <li><Link href='/disclosure'><a>Disclosure</a></Link></li>
-          <li><Link href='/tabs'><a>Tabs</a></Link></li>
+          <li><Link href="/accordion"><a href="./accordian">Accordion</a></Link></li>
```

Comment:
I'm not sure this syntax is correct - you shouldn't need the `href` tag on the `a` element if it's a child of `Link` because Next.js will generate the complete markup. Take a look at [this](https://stackoverflow.com/questions/47875730/how-can-i-fix-jsx-a11y-anchor-is-valid-when-using-the-link-component-in-react) for a possible workaround (then ask another SBXer to review that change!).